docs/icd.md

```diff
@@ -0,0 +1,72 @@
+# Interface Control Document (ICD) - `svc-cargo`
+
+<center>
+
+<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+
+</center>
+
+## Overview
+
+This document defines the gRPC and REST interfaces unique to the `svc-cargo` microservice.
+
+Attribute | Description
+--- | ---
+Status | Draft
+
+## Related Documents
+
+Document | Description
+--- | ---
+[Requirements - `svc-cargo`](https://docs.google.com/spreadsheets/d/1OliSp9BDvMuVvGmSRh1z_Z58QtjlSknLxGVdVZs2l7A/edit#gid=0) | Requirements for this module
+[Software Design Document (SDD)](./sdd.md) | 
+
+## Frameworks
+
+See the Services ICD.
+
+## REST
+
+### Files
+
+The common REST interface OAS definition is located in `svc-cargo-rest.json`.
+
+Code stubs autogenerated from the OAS definition will be located at `svc-cargo-rest.rs`.
+
+### Authentication
+
+See the Services ICD.
+
+### Endpoints
+
+| Endpoint | Type | Arguments | Description |
+| ---- | --- | ---- | ---- |
+| `/cargo/query` | GET | port_depart<br>port_arrive<br>time_range_start<br>time_range_end<br>cargo_weight_kg | Queries for a flight with the given characteristics
+| `/cargo/confirm` | PUT | flight_plan_id | Customer confirmation of a possible flight plan
+| `/cargo/cancel` | DELETE | flight_plan_id | Cancel a flight plan
+| `/cargo/region` | GET | latitude, longitude | Get vertiports for a user
+
+
+## gRPC
+
+### Files
+
+These interfaces are defined in a protocol buffer file, `svc-cargo-grpc.proto`.
+
```

Comment:
Consider adding a link to the proto file.

Reply:
Will add

Reply:
Fixed: https://github.com/Arrow-air/svc-cargo/pull/1/commits/9cdbe864c593acdc4bf89bcc76db1d2aa2e97227